ci_test/unit_tests/test_unit_subgrid_concat.py

```diff
@@ -19,7 +19,7 @@
 
 # Data
 np.random.seed(202201181)
-_num_samples = 19
+_num_samples = 32

```

Comment:
Does this need to change? These are set to odd numbers in the tests so that the batch size doesn't divide the data set size, as we've had bugs show up in the "remainder batch". If subgrid layers cannot handle this, that's a limitation we need documented.

Reply:
Comment from @aj-prime "we don’t support non power of 2 batch sizes."